linkcheck/cache/urlqueue.py

```diff
@@ -120,7 +120,9 @@ def _put (self, url_data):
         log.debug(LOG_CACHE, "queueing %s", url_data.url)
         key = url_data.cache_url
         cache = url_data.aggregate.result_cache
-        if url_data.has_result or cache.has_result(key):
+        if cache.has_result(key):
+                return
```

Comment:
The return seems to be indented twice as much as it needs to be.

(Perhaps an issue with tabs vs spaces that GitHub hides from me?)